src/lib.rs

```diff
@@ -42,11 +42,12 @@ pub use error::Error;
 pub use platform::Platform;
 pub use service::Service;
 
-pub use cbor_smol::{cbor_deserialize, cbor_serialize, cbor_serialize_bytes};
+pub use cbor_smol::{cbor_deserialize, cbor_serialize_bytes};
 pub use heapless_bytes::Bytes;
-pub use postcard::{from_bytes as postcard_deserialize, to_slice as postcard_serialize};
 
-pub fn postcard_serialize_bytes<T: serde::Serialize, const N: usize>(
+pub(crate) use postcard::from_bytes as postcard_deserialize;
+
+pub(crate) fn postcard_serialize_bytes<T: serde::Serialize, const N: usize>(
```

Comment:
In the RSA PR, I used postcard in the import API to serialize structured data as bytes for key import. Given that it makes sense for me to keep postcard as part of the public API.
Maybe it could be kept private if we make RSA key import it's own method were the Client makes the serialization transparently.

Reply:
Hmm, I’ll have to take another look at that use case.  My main motivation for dropping the `postcard` re-export is making https://github.com/trussed-dev/trussed/issues/63 feasible.